pkgs/servers/monitoring/telegraf/default.nix

```
@@ -12,10 +12,10 @@ buildGoModule rec {
     owner = "influxdata";
     repo = "telegraf";
     rev = "v${version}";
-    sha256 = "sha256-iFigbnqUXWQrhDovwQYZsr8llkB9553c5LgcKJ78yZ4=";
+    sha256 = "sha256-8shyNKwSg3pUxfQsIHBNnIaks/86vHuHN/SroDE3QFU=";
   };
 
-  vendorSha256 = "sha256-m53S/L71nyioCBbIDDAWEnqStBdqTFGq16y5ozsXq1c=";
+  vendorSha256 = "sha256-jP6P2NShzlFCptCQ04XY4cIrONNArwthiEONEo32Btw=";
```

```suggestion
  vendorSha256 = "1zx64953rscfxqmnf74vkp8kamr2i8l3jccwcj7683bpqskj9q8x";
```
Are those different for darwin?

---

Strange, but it seems as it is different for darwin. The darwin `vendorSha256` is also not valid for my x86_64 linux systems.

I don't know the reason and also cannot reproduce that as I don't have a darwin device.

---

Can you mark it broken for darwin with a comment explaining the situation?

---

It might be only soft-broken on darwin actually. Even if it does not compute the same hash itself, when the vendor derivations gets substituted, telegraf might be still build fine on macOS.

---

Let's not mark all packages broken just because there is a bug in buildGoModule. https://github.com/NixOS/nixpkgs/issues/129730

---

This is not a bug technically, but rather a case-insensitivity issue on darwin. May be possible to fix in extraPostFetch.

---

So what type of normalization do we need to do to fix this?

---

usually deleting files (if they are unnecessary) or renaming them in a way that can deal with the nix case hack suffix.

Best would ofc be to get this fixed upstream.